src/test/java/com/google/api/codegen/testdata/discoveries/ruby/ruby_cloudbilling.v1.json.baseline

```diff
@@ -77,37 +67,31 @@ end
 #    https://console.developers.google.com/apis/api/cloudbilling
 # 2. This sample uses Application Default Credentials for authentication.
 #    If not already done, install the gcloud CLI from
-#    https://cloud.google.com/sdk/ and run
+#    https://cloud.google.com/sdk and run
 #    `gcloud beta auth application-default login`
 # 3. Install the Ruby client library and Application Default Credentials
 #    library by running `gem install google-api-client` and
 #    `gem install googleauth`
+
 require 'googleauth'
 require 'google/apis/cloudbilling_v1'
 require 'json'
 
 service = Google::Apis::CloudbillingV1::CloudbillingService.new
 
-# Use Application Default Credentials for authentication when running locally.
-# For more information, see:
-# https://developers.google.com/identity/protocols/application-default-credentials
 service.authorization = \
     Google::Auth.get_application_default(['https://www.googleapis.com/auth/cloud-platform'])
 
-
-# TODO: Change placeholder below to desired parameter value for the `list_billing_account_projects` method:
-
 # The resource name of the billing account associated with the projects that you want to list. For
 # example, `billingAccounts/012345-567890-ABCDEF`.
-name = 'billingAccounts/{MY-BILLINGACCOUNT}'
-
+name = ''  # TODO: Update placeholder value.
+# ex: 'billingAccounts/my-billingAccount'
 
-# Perform the call
-projectBillingInfo = service.fetch_all(items: :projectBillingInfo) do |token|
+project_billing_info2 = service.fetch_all(items: :project_billing_info) do |token|
   service.list_billing_account_projects(name, page_token: token)
 end
 
-projectBillingInfo.each do |project_billing_info|
```

Comment:
Is this the planned name resolution mechanism for Ruby? Produces an unpleasant result in this case.